homework/01LessonOne.js

```diff
@@ -153,47 +155,47 @@ var notNotUndefined;
 
 // Problem 21
 // What is the value of 4 === 4
-var fourEqualTofour;
+var fourEqualTofour = 'true';
```

Comment:
No need to put the following Boolean values in quotation marks. Just simply return true or false.